index.js

```diff
@@ -22,16 +16,25 @@ async function run() {
 		const {stdout: t} = await execFile('git', ['tag', '-l', '--sort=-creatordate']);
 		const tags = t.split('\n').filter(Boolean).map(tag => tag.trim());
 
+		if (tags.length === 0) {
```

Comment:
Currently it doesn't seem to support creating a release for the first tag (which is usually just "Initial release"), so this condition should reflect that, at least for the time being.

```suggestion
		if (tags.length < 2) {
```

Reply:
This is unchanged. Did you add support for "initial release" in the commit?

Reply:
Support for "initial release" has been there the whole time.

If there is only one tag, then we compare that with the root commit (initial commit) we can find.

Reply:
I'd prefer this to be just `Initial release` but that works too, for the time being